src/data/nat/fib.lean

```diff
@@ -82,6 +82,10 @@ lemma fib_le_fib_succ {n : ℕ} : fib n ≤ fib (n + 1) := by { cases n; simp [f
 @[mono] lemma fib_mono : monotone fib :=
 monotone_of_monotone_nat $ λ _, fib_le_fib_succ
 
+/-- If `2 ≤ n`, `fib` is strictly monotone. -/
```

Comment:
```suggestion
/-- If `2 ≤ n`, `fib n` is strictly monotone. -/
```
I think this makes more sense...alternatively you can just say "`fib (n + 2)` is strictly monotone"...whatever you think reads better.

Reply:
Thanks! I like your shorter way. Fixed.